pymc4/_random_variables.py

```diff
@@ -44,6 +62,8 @@
     "Wishart",
 ]
 
+__all__ = tfp_supported + tfp_unsupported
```

Comment:
If we do this how will the user know which distributions they can put into a tensorflow graph and which ones they can't?

Reply:
Or do they all work because of the additional code below?

Reply:
They all work, I think that comment about pymc3 is a bit confusing, even though it makes sense once you know what is meant. @eigenfoo maybe we can reword that or just get rid of it.